.circleci/config.yml

```diff
@@ -29,11 +29,17 @@ jobs:
             - ~/.cache/pip
 
       - run:
-          name: "Run tests"
+          name: "Run local tests only "
           command: |
             . .venv/bin/activate
             export SKIP_TAGS=real; nose2
 
+      - run:
+          name: "Run all tests"
+          command: |
+            . .venv/bin/activate
+            nose2
```

Comment:
Does this mean that the CI is running the local tests twice?

Reply:
Yes it does

Reply:
Shall we not do that and have shorter build times?

Reply:
We could do, the local tests currently take <1second to run, so i don't think it's a problem right now.